fitsio/fitsio_pywrap.c

```diff
@@ -4131,6 +4131,16 @@ PyFITS_cfitsio_version(void) {
     return PyFloat_FromDouble((double)version);
 }
 
+static PyObject *
+PyFITS_cfitsio_use_standard_strings(void) {
+    if ( fits_use_standard_strings() ) {
+        Py_RETURN_TRUE;
+    } else {
+        Py_RETURN_FALSE;
+    }
+}
```

Comment:
We're going to have to figure out what to do if an older version of cfitsio doesn't have this function when linking. We might need a further compile-time check w/ a macro when building the wrapper.

Reply:
This is why the tests are failing.